### ISSUE TITLE
(Question) Does this project is similar to ZeroNet?

### ISSUE THREAD

**Issue by hardBSDk** [OkuBrowser/oku#292]
[ZeroNet](https://zeronet.io/) allow users to create and share websites as torrents of the BitTorrent protocol.

It allow the website to be more fast and efficient without additional costs (don't depend on a single server to serve the new data requests, but a network of peers hosting a part or copy of the website data), scale naturally and be censorship-resistant.

While the torrents of the BitTorrent protocol are static, the ZeroNet websites are dynamic, thus the website content can be updated anytime and the changes are sent to the peers.

These changes are coordinated using public key changes of the website owner, a comparison is done to determine if your copy of the website is old or up-to-date.

Does Oku allow this type of sharing using P2P peers?

**Comment by emmyoh** [comment]
Yes, it does! Oku replicas are mutable; when a peer tries to access a replica, it tries to retrieve the newest version. Replica IDs are public keys, so the ID does not change when the replica changes.
